src/sentry/api/paginator.py

```diff
@@ -116,10 +126,28 @@ def get_result(self, limit=100, cursor=None):
         return build_cursor(
             results=results,
             limit=limit,
+            hits=hits,
+            max_hits=max_hits,
             cursor=cursor,
             key=self.get_item_key,
         )
 
+    def count_hits(self, queryset):
```

Comment:
`queryset` argument is provided here but never actually used in favor of `self.queryset`, I don't have enough familiarity here to know which one is the correct one at this point.